src/router/bundle/bundleGenerator.ts

```diff
@@ -129,4 +129,51 @@ export default class BundleGenerator {
         response.entry = entries;
         return response;
     }
+
+    static generateBatchBundle(baseUrl: string, bundleEntryResponses: BatchReadWriteResponse[]) {
+        const id = uuidv4();
+        const response = {
+            resourceType: 'Bundle',
+            id,
+            type: 'batch-response',
+            link: [
+                {
+                    relation: 'self',
+                    url: baseUrl,
+                },
+            ],
+            entry: [],
+        };
+
+        const entries: any = [];
+        bundleEntryResponses.forEach((bundleEntryResponse) => {
+            let status = '200 OK';
+            if (bundleEntryResponse.operation === 'create') {
+                status = '201 Created';
+            } else if (
+                ['read', 'vread'].includes(bundleEntryResponse.operation) &&
+                isEmpty(bundleEntryResponse.resource)
+            ) {
+                status = '403 Forbidden';
+            } else if (bundleEntryResponse.operation === undefined || bundleEntryResponse.vid === '') {
+                status = '404 Not Found';
```

Comment:
If `fundleEntryResponse.operation` is undefined, does that mean the request operation is undefined? If so, it should not be 404.